e2e/shared/common.js

```diff
@@ -14,9 +14,11 @@ const tmp = './.e2e-tmp/';
 const cwdOpts = { cwd: tmp };
 
 const skyuxConfigPath = path.resolve(process.cwd(), tmp, 'skyuxconfig.json');
+const appExtrasPath = path.resolve(process.cwd(), tmp, 'src/app', 'app-extras.module.ts');
```

Comment:
Non-blocking comment, but the last two params could be combined into a single string.